Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -199,6 +199,63 @@ - (void)testRemoveUser {
     [self waitForExpectationsWithTimeout:60.0 handler:nil];
 }
 
+- (void)testDeviceRegistration {
+    RLMApp *app = [RLMApp appWithId:self.appId configuration:[self defaultAppConfiguration]];
+    XCTestExpectation *expectation = [self expectationWithDescription:@"should login anonymously"];
+    __block RLMUser *syncUser;
+    [app loginWithCredential:[RLMCredentials anonymousCredentials] completion:^(RLMUser *user, NSError *error) {
+        XCTAssert(!error);
```

Comment:
XCTAssertNotNil will give better error messages when the test fails than this.

Reply:
I went with `XCTAsserNil(error)` since the test should fail if there is an error.